### DISCUSSION TITLE
Crashes after the update.

### DISCUSSION THREAD

**Discussion by PalutoBaluloni** [gpx-animator/gpx-animator#508] [Q&A]
Hi. 
After upgrading to 1.8.0. 
I click the "Render" button and the program closes.
How to fix this bug?

Thanks.

**Comment by McPringle** [comment]
Sorry, I missed your question. Have you tried v1.8.2? Does it have the same problem? If yes, please provide the OS you use and how you installed GPX Animator (using the setup program or direct JAR download). Does it show any error message in the UI or command line?

**Reply by DeeDabbelju** [reply]
Hi,
I have the same problem. A .ga.xml configuration that renders correctly with version 1.7.0 crashes in version 1.8.2. I use Windows 10.0.19045.2965 and have gpx-animator installed using the setup program. There is no error message. The gui closes silently. No hint in the log-file.
please find attached an archive with the .gpx, .ga.xml and the used Taxi.png
[Tracks.zip](https://github.com/gpx-animator/gpx-animator/files/11636733/Tracks.zip)
Can you reproduce the crash?

**Reply by McPringle** [reply]
Sorry for my late reply, the last few months I took a sabbatical and rode my bicycle through the whole US. I'll check it out now.

**Reply by McPringle** [reply]
Hmmm. I downloaded your files and tried it on my computer: It works, the video was successfully created. But my main computer is running Linux. I'll setup a VM with Windows 10, but that may need a day or two.

**Reply by turfutils** [reply]
Version 1.8.2
I also have problems with crashes when I press render. I have tried different gpx-files, from strava and ridewithgps. I have tried different maps. The preview button works fine. Also played around with other settings like skip pauses.
I also tried to run with commandline and the same problem. Then I got annoyed and tried to start Virtualbox to install on Linux as you said that worked and notice not even that start. So I uninstalled everything Oracle including all java versions I had and updated the adoptium version to the latest after a few reboots. And now ... it still crashes :-)

But I thought I would try the command line once more and look! It works! Does this help for finding what is wrong? Any error logfiles you want?

**Comment by turfutils** [comment]
Did yet another purge of everything Java. Found some old java installations named Eclipse and removed them too. Downloaded Amazons openjava and now it renders fine after starting normally!

Can't answer for the original poster but mine is working now.
